integration/inspektor-gadget/integration_test.go

```diff
@@ -503,10 +504,35 @@ func TestEbpftop(t *testing.T) {
 	t.Parallel()
 
 	ebpftopCmd := &Command{
-		Name:           "StartEbpftopGadget",
-		Cmd:            fmt.Sprintf("$KUBECTL_GADGET top ebpf"),
-		ExpectedRegexp: fmt.Sprintf(`\S*\s+\d+\s+Tracing\s+ig_top_ebpf_it\s+\d+\s+\S*\s+`),
-		StartAndStop:   true,
+		Name:         "StartEbpftopGadget",
+		Cmd:          "$KUBECTL_GADGET top ebpf -o json",
+		StartAndStop: true,
+		ExpectedOutputFn: func(output string) error {
+			expectedEntry := &ebpftopTypes.Stats{
+				Name: "ig_top_ebpf_it",
+				Type: "Tracing",
+			}
+
+			normalize := func(e *ebpftopTypes.Stats) {
+				e.Node = ""
+				e.Namespace = ""
+				e.Pod = ""
+				e.Container = ""
+				e.Namespace = ""
```

Comment:
Another confirmation that we need to redefine these structs because these fields don't make sense for this gadget.